portal-api/api-tests/v1/users/post-create-user.api-test.js

```diff
@@ -0,0 +1,238 @@
+const databaseHelper = require('../../database-helper');
```

Comment:
New tests added on `loc 115 - loc 154` -- I've just pulled this file out into it's own thing so we don't have such a giant `users.api-test.js`. Naming of file follows existing pattern for api-tests